index.js

```diff
@@ -76,19 +78,26 @@ if (!output && !dir && !argv.replace) {
 // Use warn to avoid writing to stdout
 console.warn(chalk.bold.red(logo))
 
-spinner.text = `Loading Config`
+spinner.text = 'Loading Config'
 spinner.start()
-Promise.all([ globber(input), getConfig({}, argv.config) ])
-  .then((arr) => {
-    // Until parameter destructuring is supported
-    let files = arr[0]
+getConfig({}, argv.config)
```

Comment:
Maybe it is better/necessary to move config loading into processCSS to expose the current file to `postcss.config.js`and load the config **per file**

```js
module.exports = ({ options, file, env }) => ({ // { options, file, env } === ctx && options = argv.ctx
  parser: file.extname === '.sss' ? 'sugarss' : false
  map: options.map || false
  plugins: {
     'postcss-modules':  options.modules ? {} : false
     'cssnano':  env === 'production' ? options.minify : false
  }
})
```